src/tests/multihost/ad/test_adparameters_ported.py

```diff
@@ -30,17 +30,19 @@ def fixture_create_plain_aduser_group(session_multihost, request):
     # Create user
     session_multihost.client[0].run_command(
         f'adcli create-user {ad_user} -D {domainname} --display-name='
-        f'"Plain {ad_user}"', stdin_text=password, raiseonerr=False
+        f'"Plain {ad_user}" --stdin-password -v',
+        stdin_text=password, raiseonerr=False
     )
     # Create group
     session_multihost.client[0].run_command(
         f'adcli create-group {ad_group} -D {domainname} -z '
-        f'"Plain {ad_group}"', stdin_text=password, raiseonerr=False
+        f'"Plain {ad_group}" --stdin-password -v',
+        stdin_text=password, raiseonerr=False
     )
     # Add member
     session_multihost.client[0].run_command(
-        f'adcli add-member -D {domainname} {ad_group} {ad_user}',
-        stdin_text=password, raiseonerr=False
+        f'adcli add-member -D {domainname} {ad_group} {ad_user} '
+        f' --stdin-password -v', stdin_text=password, raiseonerr=False
```

Comment:
Hi,

thanks for the fix. There is another similar call on line 2853 and there are also similar calls in `test_adparameters.py`. I'm not sure if the related tests are currently run but it might be worth to fix all places to avoid issues in future.

bye,
Sumit

Reply:
Good catch I did not get to review test_adparameters.py yet as they are run in different job.
Hopefully all occurences are fixed now.